samples/panther/ap/src/ipm.c

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright (c) 2017 IpTronix S.r.l.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <stdio.h>
+#include <ipm.h>
+#include <ipm/ipm_quark_se.h>
+#include <sensor.h>
+#include <zephyr.h>
+#include <misc/printk.h>
+
+#include <mqtt.h>
+
+#include "mqtt.h"
+
+#define STACKSIZE               2000
+
+#define MAIN_FIBER_PRI          2
+
+struct shield_sens {
+	struct sensor_value BME280[3];
+	struct sensor_value ADXL362[4];
+	struct sensor_value ADXRS290[3];
+	struct sensor_value SI1153[5];
+};
+
+QUARK_SE_IPM_DEFINE(ess_ipm, 0, QUARK_SE_IPM_INBOUND);
+
+static char ipm_stack[STACKSIZE];
+static char json[2048];
```

Comment:
No need for `json` to be global.  Define it where it's used, or better yet: use the JSON library that comes with Zephyr... it's safer.